### ISSUE TITLE
Adding a Custom Location takes the Site offline

### ISSUE THREAD

**Issue by pwfraley** [jc21/nginx-proxy-manager#929]
**Are you in the right place?**
- Yes

**Checklist**
- Have you pulled and found the error with `jc21/nginx-proxy-manager:latest` docker image? Yes
- Are you sure you're not using someone else's docker image? Yes
- If having problems with Lets Encrypt, have you made absolutely sure your site is accessible from outside of your network? Yes

**Describe the bug**
- I have a very simple Site setup, which proxies to a quasar app (Works).  The only thing I have enabled (though I tried all settings) is to force ssl
- Now everything works fine.  I can reach my site under the domain name.  SSL is enabled, http requests get redirected to https
- Now I edit the Proxy Host and add a custom location /api
- Upon Saving my changes the site is taken offline

**To Reproduce**
1. Create a Simple Proxy Host with SSL (Lets Encrypt Enabled)
2. After Certificate is setup make sure the site is reachable
3. Now Edit the proxy host and add a simple Custom location (/api)
4. Site is taken offline

**Expected behavior**
The Site stays online and is reachable.  The /api Url is also reachable

**Operating System**
- Current Debian as Docker host

**Additional context**
Basically I am trying to host a modern SPA Website.  The Frontend is created using quasar/vue 3 and the Backend, which is a simple rest api is done using Dotnet Core C#.  I want to host the quasar App under / and the api under /api.

**Comment by l4rm4nd** [comment]
Your custom location basically broke the nginx config somehow. Take a look at the config itself for your proxy host and try to understand, why its malformed. Maybe you can find some helping infos in the logs.

**Comment by pwfraley** [comment]
I figuered it out.  When I first added the proxy host I had a typo in the backend service.  Fixing the typo did not fix it in the config.  I had to delete the entire proxy host and then recreate it.  Then all worked fine.

**Comment by dscpt** [comment]
There's still something not right here, I have the same issue...

When a custom location is added there is no conf file generated and the proxy host shows as offline.

I was able to work around it by ignoring the gui and adding the extra location into the advances box on the proxy site tab.

![image](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/87764256/fd3a7b4f-2268-4868-a8cd-3ef6d467ef53)

**Comment by MeesJ** [comment]
After I got a new installation of NPM I started having this issue as well. Are there any workarounds that don't require messing around with the configuration files?

**Comment by b1nary-b0b** [comment]
I have the same issue.

**Comment by redsky-air** [comment]
This problem can only be said to have arisen again!

**Comment by jricardooliveira** [comment]
Same issue here too... This used to work and it's not working in the latest version.

**Comment by AkshayRao27** [comment]
See if https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3512#issuecomment-1940868886 works for you?

**Comment by hans-holgersson** [comment]
I have the same problem with "custom location".
I am using NPM in docker.
This trick (https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3512#issuecomment-1940868886) solves the problem - at least for now.
10x.

**Comment by washingtonsousa** [comment]
I'm having the same problem here, workaround overwriting the gui using raw text solved the problem, on container logs I've seen erros of trying to delete files on directory.

Its so impressive sometimes a pull request that doest change anything can stop working something that never was a problem before...

**Comment by pageagency** [comment]
Same error. It looks like it was introduced on the last update? Is there a real solution for this?

**Comment by AkshayRao27** [comment]
Till there's an update and a real fix, https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3512#issuecomment-1940868886 seems to be a workaround that's fixing it for several people.

**Comment by hans-holgersson** [comment]
hi ... has anyone had a chance to work on this issue?

**Comment by tkbstudios** [comment]
It's a user issue. If you make it correctly, it won't take it offline.

**Comment by hans-holgersson** [comment]
> It's a user issue. If you make it correctly, it won't take it offline.

probably you are right. In this case tell me please where is the the error, which causes the troubles in the folowing setup:

![image](https://github.com/user-attachments/assets/29d22aea-b341-470b-bbbb-3f7a2a9f3736)

**Comment by hans-holgersson** [comment]
So, to be clear ...

I want to instantiate a location in order to address https://www.site.com/personal to be served from the root of the webservice runnig on host 192.168.0.1 on tcp port 81.

This is what i am trying to implement, but when i save the config the site goes offline.

How could i do this to work?

**Comment by pageagency** [comment]
I don't know about personal but I've had the same issue with /admin and gave up.

**Comment by hans-holgersson** [comment]
i will give it a try with the latest npm ...
